Resources/Prototypes/Datasets/Names/books.yml

```
@@ -0,0 +1,246 @@
+- type: dataset
+  id: book_type
+  values:
+  - book
+  - folio
+  - collection
+  - notes
+  - manuscript
+  - records
+  - tome
+  - journal
+  - archives
+  - codex
+  - memories
+  - compendium
+
+- type: dataset
+  id: book_genre
+  values:
+  - detective
+  - comedy
+  - horror
+  - poetry
+  - novella
+  - chronicle
+  - science-fiction
+  - fantasy story
+  - romance
+  - thriller
+  - historical fiction
+  - biography
+  - adventure
+  - drama
+
+- type: dataset
+  id: book_hint_appearance
+  values:
+  - ancient
+  - shabby
+  - dirty
+  - unusual
+  - faded
+  - nasty
+  - dusty
+  - scary
+  - bloody
+  - bright
+  - dubious
+  - intriguing
+  - ugly
+  - crooked
+  - crumpled
+  - dirty
+  - elegant
+  - ornate
+  - weathered
+  - chrisp
+  - lavish
+  - tattered
+  - polished
+  - embossed
+  - mismatched
+  - gilded
+  - strange
+
+- type: dataset
+  id: book_character
+  values:
+  - clown
+  - mime
+  - reporter
+  - butcher
+  - bartender
+  - janitor
+  - engineer
+  - scientist
+  - guard
+  - doctor
+  - chemist
+  - prisoner
+  - researcher
+  - trader
+  - captain
+  - unath
+  - niana
```

```suggestion
  - lizard
  - moth
```
The upstream doesn't use the Paradise species' names.